lib/Client.php

```diff
@@ -353,7 +354,7 @@ protected function doRequest(RequestInterface $request): ResponseInterface
      * Has a list of curl handles, as well as their associated success and
      * error callbacks.
      *
-     * @var array<int, mixed>
+     * @var array<int, array{0: RequestInterface, 1: callable, 2: callable, 3: int}>
```

Comment:
This at least helps to document what this array-of-arrays should look like.
But I locally tried specifying wrong things here, and I couldn't get `phpstan` to give any errors - I was expecting that lines of code that set and/or use entries in this array would cause `phpstan` to complain.